docs/source/menupage.rst

```diff
@@ -195,7 +195,7 @@ Wagtail has a restriction that forbids models from subclassing more than one oth
 
 .. _manipulating_submenu_items:
 
-Using ``MenuPage`` to manipulating sub-menu items
+Using ``MenuPage`` to manipulate sub-menu items
 =================================================
```

Comment:
@DanAtShenTech Tiny thing, but can we knock a couple of underline characters from here to match the heading length?

Reply:
Fixed now @ababic